modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,53 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Add the `BUNDLE_GEN_FLAGS` variable definition to your `Makefile` file and make the following changes:
++
+.`Makefile` diff
+[source,diff]
+----
+...
+-  $(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
++  $(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
++  $(BUNDLE_GEN_FLAGS)
+
+BUNDLE_GEN_FLAGS ?= -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+
+# USE_IMAGE_DIGESTS defines if images are resolved via tags or digests
+# You can enable this value if you would like to use SHA Based Digests
+# To enable set flag to true
+USE_IMAGE_DIGESTS ?= false
+ifeq ($(USE_IMAGE_DIGESTS), true)
+	BUNDLE_GEN_FLAGS += --use-image-digests
+endif
```

Comment:
@ryantking Is this correct?

Reply:
Also, do we need the second step where the `make bundle USE_IMAGE_DIGESTS=true`? Couldn't we just set `USE_IMAGE_DIGESTS ?= false` to `true` in line 45?

Reply:
The "make bundle USE_IMAGE_DIGESTS=true" step should be the last step.